### PR TITLE
format project using black, isort and autoflake

### PR DESCRIPTION
**Pull request by rakeshgm**
Formatted the repo with black
sorted the imports with isort and removed unused imports using autoflake 

Also adding tox.ini to check the code cleanliness at the gate. 
this will run black and isort
the checks will fail, if the code is not as per these standards.

please run the below from the root your dir. 
1. black . 
2. isort -c 

submit the PR once these as successfully passed.   

Signed-off-by: rakeshgm <rakeshgm@redhat.com>

### CODE REVIEW

File: .isort.cfg [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[settings]
+multi_line_output = 3
+include_trailing_comma = True
+force_grid_wrap = 0
+use_parentheses = True
+ensure_newline_before_comments = True
+line_length = 88
```
Comment (psathyan) [inline .isort.cfg:7]: It's a good practice to end with a newline character

File: README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+Various QE scripts written by QE for Ceph Testing. This scripts once tested can be ported to main ceph-qa-scripts git repo
```
Comment (psathyan) [inline README.md:1]: We could rework... like
```
This repository contains scripts for testing Ceph. We recommend porting the scripts to `ceph-qa-scripts` once it has been found to be reliable and passes the established criteria
```

File: .isort.cfg (hunk 1 above)
Comment (rakeshgm) [reply .isort.cfg]: added newline char

**Review by psathyan — APPROVED** [review]
Let's get it merged

File: README.md (hunk 2 above)
Comment (rakeshgm) [reply README.md]: changed added

**Pull request merged**